cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -350,15 +351,16 @@ private void addSteps(
 
         for (int i = 0; i < alphas.size(); i++) {
             double alpha = alphas.get(i);
+            double alphaRatio = -alpha;
             double rho =  rhos.get(i);
             double x = 0.0;
             if (!xStepRangeIsConsistent || alphaMax == 0) {
                 x = tx.getX();
             } else {
                 if (asymmetrical()) {
-                    x = getStepXforAsymmetrical(xStepMin, xStepMax, alpha, alphaMax, theta);
+                    x = getStepXforAsymmetrical(xStepMin, xStepMax, alphaRatio, alphaMax, theta);
```

Comment:
The sign changes are a bit confusing for me and I wonder if the best way to handle this bug is with your solution or if it is maybe better to change the functions ````getStepXforAsymmetrical/Symetrical```` and assume that they are in IIDM convention ?